### PR TITLE
[WWST-2483] Add IKEA Floalt Panel to zll-dimmer-bulb.groovy

### PR DESCRIPTION
**Pull request by MGoralczykS**
@greens 
Could you review this? 
RAW: 01 C05E 0220 02 09 0000 0003 0004 0005 0006 0008 0300 0B05 1000 04 0005 0019 0020 1000

### CODE REVIEW

File: devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy [hunk 1]
```diff
@@ -46,7 +46,7 @@ metadata {
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008", outClusters: "0019", manufacturer: "innr", model: "RB 145", deviceJoinName: "Innr Smart Candle White"
 		//IKEA TRADFRI bulb E14 W op/ch 400lm: 01 C05E 0100 02 08 0000 0003 0004 0005 0006 0008 0B05 1000 04 0005 0019 0020 1000
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E14 W op/ch 400lm", deviceJoinName: "IKEA TRADFRI LED Bulb"
-
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "FLOALT panel WS 30x30", deviceJoinName: "IKEA FLOALT Panel"
```
Comment (tpmanley) [inline devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy]: From the product description it appears this device should support color temperature, plus it has cluster 0x0300

**Comment by MGoralczykS** [comment]
@tpmanley @greens 
I changed DTH. Could you review this?

File: devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy [hunk 2]
```diff
@@ -38,6 +38,7 @@ metadata {
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 1000", outClusters: "0019", manufacturer: "Philips", model: "LTW013", deviceJoinName: "Philips Hue White Ambiance Spot"
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 1000", outClusters: "0019", manufacturer: "Philips", model: "LTW014", deviceJoinName: "Philips Hue White Ambiance Spot"
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 1000", outClusters: "0019", manufacturer: "Philips", model: "LTW015", deviceJoinName: "Philips Hue White Ambiance A19"
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "FLOALT panel WS 30x30", deviceJoinName: "IKEA FLOALT Panel", mnmn: ""
```
Comment (tpmanley) [inline devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy]: Not sure if it was done on purpose but the `mnmn: ""` at the end shouldn't be necessary

**Comment by MGoralczykS** [comment]
@greens  @tpmanley  
Could you review this, one more time :)?

**Review by tpmanley — APPROVED** [review]

**Pull request merged**